docs/Deployment.md

```diff
@@ -0,0 +1,26 @@
+### Preparations
+To deploy and run the FHIR analytics pipeline agent, you will need a FHIR server instance (suggest testing with OSS instance without authentication. For paas instances, make sure you have premission to assign the Managed Identity authentication).
+
+### Deployment
+1. Go to [Custom deployment](https://ms.portal.azure.com/#create/Microsoft.Template) on Azure, them select **Build your own template in the editor**.
+2. Use provided [template](../deploy/templates/FhirSynapsePipelineTemplate.json) to deploy an analytics pipeline.
+
+![image](./assets/template_parameters.png)
+
+
+Parameters:
+- **App Name**: azure function name.
+- **Fhir Server Url**: fhir service endpoint. Managed FHIR service is recommended. You can try test with your PaaS fhir server.
+- **Authentication**: whether to access Azure API for FHIR with managed identity authentication. Set false if you are using an OSS public FHIR server instance.
+- **Fhir version**: currently R4 is supported.
+- **Data Start**: start time stamp to export the data.
+- **Data End**: End timestamp to export the data, leave it empty if you want to periodically export data in real time.
+- **Container name**: container in azure storage to store results. The Azure storage will be created by the arm template. You can find the output after deploy completes.
```

Comment:
Nit: Capitalize ARM, since it is an abbreviation.

Reply:
Sure, updated.